crates/hir/src/semantics/source_to_def.rs

```diff
@@ -343,6 +343,23 @@ impl SourceToDefCtx<'_, '_> {
         Some((container, label_id))
     }
 
+    pub(super) fn label_ref_to_def(
+        &mut self,
+        src: InFile<&ast::Lifetime>,
+    ) -> Option<(DefWithBodyId, LabelId)> {
+        let break_or_continue = self
+            .ancestors_with_macros(src.syntax_ref(), |_this, ancestor| {
+                ancestor.map(ast::Expr::cast).transpose()
+            })?;
```

Comment:
There is no need to check for macros here is there? You can't have a macro call expand to a label, that is `break label!() value` is invalid afaik, so a simple parent check should suffice here

Reply:
Oh I thought this is possible but yeah, apparently it is not.